src/docs/asciidoc/nav.adoc

```diff
@@ -0,0 +1,9 @@
+* xref:dedication.adoc[]
```

Comment:
Is [`colophon.adoc`](https://github.com/mraible/infoq-mini-book/blob/main/src/docs/asciidoc/chapters/colophon.adoc) missing from this file on purpose?

Reply:
No reason, I can add it

Reply:
Done!